google_cal_to_asana/program.py

```diff
@@ -0,0 +1,52 @@
+"""This program creates a new Asana task when a new event is created in Google Calendar.
+
+Note:
+    The `asana_client` from AutoKitteh is used for handling authentication with Asana.
+    This wraps necessary configurations for connecting to the Asana API securely.
+    However, all functional API calls, such as creating tasks, are made using
+    the official Asana client library (`asana`).
+"""
+
+import os
+
+from autokitteh.asana import asana_client
+import asana
+from asana.rest import ApiException
+
+PROJECT_GID = os.getenv("PROJECT_GID")
+
+api_client = asana_client("asana_conn")
+tasks_api_instance = asana.TasksApi(api_client)
```

Comment:
```suggestion
tasks_api_instance = asana.TasksApi(asana_client("asana_conn"))
```

Reply:
I disagree. While it may look cleaner, I believe mimicking [Asana’s basic examples](https://docs.autokitteh.com/integrations/asana) can be useful to demonstrate that there’s no need to do things differently.